docs/source/recipes/appextras.md

```diff
@@ -23,14 +27,16 @@ Or you can use the new key of `config.settings`, the `appExtras`. This is
 a list of registrations, each registration is an object with:
 
 - `match`: The `match` key is for objects compatible with [react-router's
-matchPath](https://v5.reactrouter.com/web/api/matchPath), so it can be either
-a simple string or a match object.
+  matchPath](https://v5.reactrouter.com/web/api/matchPath), so it can be either
+  a simple string or a match object.
+- `ignore`: The `ignore` key is for routes compatible with the match property from above and
```

Comment:
I'm not certain about the wording being 100% compliant with what the code actually does, so someone should double-check both the `ignore` and `match` properties rewording.

```suggestion
- `ignore`: The `ignore` key is for routes compatible with the [react-router's `matchPath`](https://v5.reactrouter.com/web/api/matchPath) property, and ignores all the results in the `ignore` rule.
  It can be either a string or a [`match`](https://v5.reactrouter.com/web/api/match) object.
```